bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/ResumingStreamingResultScanner.java

```diff
@@ -87,25 +103,82 @@ public FlatRow next() throws IOException {
       try {
         FlatRow result = currentDelegate.next();
         if (result != null) {
-          retryHandler.update(result);
+          requestManager.updateLastFoundKey(result.getRowKey());
+          // We've had at least one successful RPC, reset the backoff and retry counter
+          currentErrorBackoff = null;
+          timeoutRetryCount = null;
```

Comment:
Please correct me if I'm wrong, but doesn't ReadRowsRequestManager get created once per scan attempt whose response would contain many rows? If so, isn't the 75 ns negligible with respect to network latency and protobuf parsing